### ISSUE TITLE
Bug: Cannot use HTML in setEventMessages

### ISSUE THREAD

**Issue by marcosgdf** [Dolibarr/dolibarr#5145]
Version: 3.9.1

``` php
setEventMessage('Click <a href="http://">here</a>for more information', 'errors');
```

Expected: Message with link
Actual behaviour: Message without the link

**Comment by bafbes** [comment]
I think that the solution is that the dol_htmloutput_mesg function which is responsible of outputting the message, should be extended to handle the case where the message contains a link.

**Comment by eldy** [comment]
It seems that this is fixed in current branch 3.9
Can you confirm ?

**Comment by marcosgdf** [comment]
Yes, it is fixed now.

Thanks.
